### PR TITLE
re-worked on #Add pylint workflow and fix all issues"

### PR DESCRIPTION
**Pull request by sigmadawg**
addresses #262

### CODE REVIEW

File: apps/data_handler/handler_tools/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"""Module docstring placeholder."""
```
Comment (djeck1432) [inline apps/data_handler/handler_tools/__init__.py:1]: It won't work this way, it should contain description of the file not just some random text

File: apps/data_handler/handler_tools/api_connector.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+"""Module docstring placeholder."""
```
Comment (djeck1432) [inline apps/data_handler/handler_tools/api_connector.py:1]: It won't work this way, it should contain description of the file not just some random text

File: apps/data_handler/handler_tools/constants.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
+"""Module docstring placeholder."""
```
Comment (djeck1432) [inline apps/data_handler/handler_tools/constants.py:1]: It won't work this way, it should contain description of the file not just some random text

File: apps/data_handler/handler_tools/constants.py [hunk 4]
```diff
@@ -6,6 +8,7 @@
 # Temporary solution.
 # TODO remove it when all protocols with interest rate models will be available
 class AvailableProtocolID(Enum):
+    """class docstring"""
```
Comment (djeck1432) [inline apps/data_handler/handler_tools/constants.py:11]: It won't work this way, it should contain description of the class not just some random text

**Review by djeck1432 — CHANGES_REQUESTED** [review]
It won't work this way with some text in dostring, it should contain description of the class/file/method/function not just some random text

**Comment by sigmadawg** [comment]
@djeck1432 Could you kindly check if its alright now?

File: apps/data_handler/db/schemas.py [hunk 5]
```diff
@@ -6,7 +8,7 @@
 
 
 class LoanStateBase(BaseModel):
-    protocol_id: str
```
Comment (djeck1432) [inline apps/data_handler/db/schemas.py:9]: Why did you remove this field?

**Review by djeck1432 — CHANGES_REQUESTED** [review]
please, resolve merge conflict and revert removed field

**Comment by sigmadawg** [comment]
KIndly check

**Comment by sigmadawg** [comment]
KIndly check

**Review by djeck1432 — APPROVED** [review]
THe biggest PR ever 116 files
looks good to me

**Comment by djeck1432** [comment]
@sigmadawg Please fix one test case and I will merge it

**Comment by sigmadawg** [comment]
Kindly check
image

**Comment by sigmadawg** [comment]
is this okay?

**Comment by djeck1432** [comment]
@sigmadawg run pipeline, will see soon

**Comment by sigmadawg** [comment]
I ran the workflow, attached the pictures above

**Comment by djeck1432** [comment]
@sigmadawg Pylint not passed
https://github.com/CarmineOptions/derisk-research/actions/runs/11626963851/job/32379739187?pr=269

**Comment by djeck1432** [comment]
did you sync with `master`? because I have another result
Screenshot 2024-11-01 at 10 14 12

**Comment by sigmadawg** [comment]
Kindly check @djeck1432

**Comment by djeck1432** [comment]
@sigmadawg merged, passed

**Comment by sigmadawg** [comment]
thankss!

**Pull request merged**